nx-X11/config/cf/Imake.tmpl

```diff
@@ -735,7 +735,7 @@ TCLIBDIR = TclLibDir
 #define AlternateUsrLibDir YES
 #endif
 #else
-#define UsrLibDir Concat4(/usr/local,/,LibDirName,/nx/X11)
+#define UsrLibDir Concat4(/usr,/,LibDirName,/nx/X11)
```

Comment:
This change looks unrelated. As if the debian/patches/ files had been applied when creating this PR branch...